app/feedstock/[id]/store.tsx

```diff
@@ -0,0 +1,100 @@
+import { TooltipWrapper } from '@/components/tooltip-wrapper'
+import { Store } from '@/types/types'
+import { Column, Row } from '@carbonplan/components'
+import React, { useState } from 'react'
+import { Box, Flex, Text } from 'theme-ui'
+import { CodeSnippet } from './code-snippet'
+import { DataViewer } from './data-viewer'
+import { DatasetRepr } from './dataset-repr'
+
+export const FeedstockStore: React.FC<{ store: Store; color: string }> = ({
+  store,
+  color,
+}) => {
+  const [tooltipExpanded, setTooltipExpanded] = useState(false)
+
+  const tooltipContent = !store.public
+    ? 'Access requires credentials or a Columbia-LEAP JupyterHub server.'
+    : store.geospatial
+      ? ''
+      : 'This dataset contains non-geospatial data not supported by the data viewer.'
+
+  return (
+    <Box sx={{ ml: [3, 4, 5, 6] }}>
+      <Row columns={[1, 1, 12, 12]}>
```

Comment:
I think the grid setup here is creating some elements to overrun the 6 columns on mobile and 8 columns on tablet:
![CleanShot 2024-10-03 at 17 18 37@2x](https://github.com/user-attachments/assets/1abe759f-90eb-4661-8e40-e3defe35ee90)